webpack.config.js

```diff
@@ -113,6 +114,9 @@ const wordpressExternals = ( context, request, callback ) =>
  */
 // eslint-disable-next-line no-unused-vars
 function getWebpackConfig( { externalizeWordPressPackages = false } = {}, argv ) {
+	const cssFilename =
+		isDevelopment || calypsoEnv === 'desktop' ? 'style-[name].css' : 'style-[name].[chunkhash].css';
```

Comment:
With this when we build a Gutenberg extension from the SDK for production, the output CSS files will contain the chunkhash in the name:

```
NODE_ENV=production npm run sdk:gutenberg -- --editor-script=client/gutenberg/extensions/editor-notes/index.js
```

Perhaps it's a good idea to keep it only `style-[name].css` for the SDK, regardless of whether we're building for development or production.